packages/dcos-history/extra/history/server.py

```diff
@@ -90,4 +92,10 @@ def start():
 
     state_buffer = BufferCollection(os.environ['HISTORY_BUFFER_DIR'])
     BufferUpdater(state_buffer, headers_cb).run()
-    app.run(host='0.0.0.0', port=int(os.environ.get('PORT', '15055')))
+
+
+def start():
+    # Used for testing only
+    # Have to be in the same folder to run this
+    # In case of failure it will not sys.exit
+    os.system("/opt/mesosphere/bin/gunicorn -c dcos_history_conf.py server:app")
```

Comment:
@amitaekbote would still be nice if we can just say `gunicorn`. For users doing testing inside a DC/OS cluster that should be in $PATH. It hsould also be so if people are in a virtualenv and have done a `pip install -e`.